SPECS/net-snmp.spec

```diff
@@ -520,20 +411,44 @@ rm -rf ${RPM_BUILD_ROOT}
 %dir %{_localstatedir}/lib/net-snmp/cert_indexes
 
 %files agent-libs
-%{_libdir}/libnetsnmpagent*.so.*
-%{_libdir}/libnetsnmphelpers*.so.*
-%{_libdir}/libnetsnmpmibs*.so.*
-%{_libdir}/libnetsnmptrapd*.so.*
+%{_libdir}/libnetsnmpagent*.so.%{soname}*
+%{_libdir}/libnetsnmphelpers*.so.%{soname}*
+%{_libdir}/libnetsnmpmibs*.so.%{soname}*
+%{_libdir}/libnetsnmptrapd*.so.%{soname}*
+
+%changelog
+* Wed Aug 13 2025 Gaëtan Lehmann <gaetan.lehmann@vates.tech> - 5.9.3-8.1
+- Rebase on 5.9.3-8
+- Remove obsolete net-snmp-5.7.2-CVE-2022-24806.patch
+- *** Upstream changelog ***
+  * Mon Jan 20 2025 Deli Zhang <deli.zhang@cloud.com> - 5.9.3-8
+  - CP-50295: Add epoch 1 for XS 8
```

Comment:
I note that this version is linking to ssl3:

```
rpm -qRi ~/Downloads/net-snmp-5.9.3-8.xs8.x86_64.rpm  | grep ssl
libssl.so.3()(64bit)
```

May we confirm it in downstream CL